script/test

```diff
@@ -298,11 +311,22 @@ do_build_otbr_docker()
 
     if [[ ${NAT64} == 1 ]]; then
         otbr_options+=("-DOT_NAT64_BORDER_ROUTING=ON")
+        if [[ ${NAT64_SERVICE} == "openthread" ]]; then
+            otbr_options+=("-DOT_NAT64_TRANSLATOR=ON")
+        fi
+        args+=("NAT64_SERVICE=${NAT64_SERVICE}")
     else
         otbr_options+=("-DOT_NAT64_BORDER_ROUTING=OFF")
     fi
```

Comment:
How about change the default value of `NAT64_SERVICE` to `openthread` in ot-br-posix/etc/docker/Dockerfile, and delete the NAT64 assignment here? Otherwise we need to maintain the sync of options here with those in ot-br-posix (https://github.com/openthread/ot-br-posix/blob/main/script/_otbr#L113-L119).